### ISSUE TITLE
Hills workshop: confirm facilitator 

### ISSUE THREAD

**Issue by larahanlon2** [carbon-design-system/ibm-dotcom-library#1647]
DDS team is planning to write Hills. In order to successfully create Hills together, we need research and alignment around core findings and artifacts.

## Tasks

- [x] Lara to reach out to EDT team and network to find a facilitator
- [ ] Lara to work with the facilitator to educate them on our team
- [ ] Lara to co-ordinate agenda with facilitator 

## Acceptance criteria

- The facilitator has a Coach badge at minimum
- The facilitator is available March 25–26

**Comment by larahanlon2** [comment]
Reached out to Jessica Tremblay on March 3rd with request. She suggested [Nina Kryza](https://w3.ibm.com/bluepages/profile.html?uid=3G0636897#historyCarousel) – OM for EDT program. Waiting for her to confirm her availability.

**Comment by larahanlon2** [comment]
@ljcarot @wonilsuhibm @jeffchew @RobertaJHahn 

Unfortunately, the facilitator I was in touch with (and who thought they had bandwidth) can no longer commit to the workshop due to shift in priorities on her team end of last week. I'm following up with one other lead. If that fails by EOD, I will ask Nigel for a referral as he volunteered to help.

**Comment by larahanlon2** [comment]
@ljcarot @wonilsuhibm @jeffchew @RobertaJHahn 

Met with [Emery Ford](https://w3.ibm.com/bluepages/profile.html?uid=1J9276897) today to discuss the request. She's extremely experienced and has excellent credibility. She is available and interested for March 25–26 but as she is in GBS her hours are billable. Waiting for a pricing estimate.

**Comment by larahanlon2** [comment]
Although we cannot fund Emery's time, she has agreed – with manager approval – to facilitate our workshop for free! Caveat: with the current covid-19 climate there is a concern that this is not the right time for Hills workshop. In order for this to be a success, we need full team participation, a clear mission statement, and stakeholder alignment. 

Waiting on confirmation from DDS Leadership whether to move ahead or not.

**Comment by larahanlon2** [comment]
Hills workshop postponed. Considering this issue complete for now.
